### DISCUSSION TITLE
peering external to our database

### DISCUSSION THREAD

**Discussion by demimike** [gruntwork-io/knowledge-base#581] [How do I] [answered]
We are trying to use Hasura cloud with our internal database. The peering has been set up. I have the vpc-peering-external to allow connections between the remote subnet and the app vpc. I added the remote address to the database security group. I'm not able to connect to the database yet. Is there anything else we have to configure to get this to work in the ref arch? Should I be connecting through some other VPC?

---

<ins datetime="2022-11-02T14:42:22Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109513">Tracked in ticket #109513</a></p>
</ins>

**Comment by josh-padnick** [comment, marked as answer]
What you describe should be sufficient, but diagnosing networking issues is always tricky, so here are a few tips:

1. Amazon just released the [VPC Reachability Analyzer](https://docs.aws.amazon.com/vpc/latest/reachability/what-is-reachability-analyzer.html). Perhaps that works across account VPCs?
2. The most common culprits with network reachability issues are Network ACLs, subnet routing configurations, security groups, and Internet gateway routes. Make sure to check on the configuration for each of those.
